### DISCUSSION TITLE
Using migrate and studio in serverless enviroment

### DISCUSSION THREAD

**Discussion by MinimogDev** [prisma/prisma#3221] [Q&A] [answered]
I'm setting up prisma to run on [aws lambda](https://aws.amazon.com/lambda/) and use [serverless aurora database ](https://aws.amazon.com/rds/aurora/serverless/) Postgresql version.

There are 2 areas I am not able to address atm and that is:

1. How can I use prisma migrate in serverless environment
2. How can I use prisma studio from my machine to connect to these databases and view data, especially since aws databases are encrypted by default.

**Comment by ryands17** [comment, marked as answer]
Hey @MinimogDev 👋 

> 1. How can I use prisma migrate in serverless environment

Prisma Migrate can be easily run in a serverless environment. If you want to do it programatically, you would need to use `exec` to run the Prisma binary. Another point to note is that you would need the binaries for the AWS Lambda environment (`rhel-openssl-1.0.x`) mentioned [here](prisma.io/docs/reference/tools-and-interfaces/prisma-schema/generators#supported-operating-systems) as that is required for migrate. Are you facing any issues there?

> 2. How can I use prisma studio from my machine to connect to these databases and view data, especially since aws databases are encrypted by default.

Unfortunately, Studio runs on websockets, so it would take a whole lot of config to run Studio on Lambda. I personally don't have any experience of this but if you still require it, best to open a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for serverless support.

Let me know if this helps :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
